llvm/lib/ProfileData/InstrProfWriter.cpp

```diff
@@ -521,6 +532,18 @@ static Error writeMemProfV0(ProfOStream &OS,
   return Error::success();
 }
 
+// Write out MemProf Version1 as follows:
+// uint64_t Version
```

Comment:
consider adding something like "NEW in v1" or something here and similar for the new in v2 fields there.

Reply:
Thank you for the review!  Added "NEW" comments.